wasm/solana-client/src/methods/token_largest_accounts.rs

```diff
@@ -45,21 +45,10 @@ impl From<GetTokenLargestAccountsRequest> for ClientRequest {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
-#[serde(rename_all = "camelCase")]
-pub struct TokenLargestAccountsValue {
-    #[serde(deserialize_with = "deserialize_public_key")]
-    pub address: Pubkey,
-    pub amount: String,
-    pub decimals: u8,
-    pub ui_amount: Option<u64>,
-    pub ui_amount_string: String,
-}
-
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct GetTokenLargestAccountsResponse {
     pub context: Context,
-    pub value: Vec<TokenLargestAccountsValue>,
+    pub value: Vec<RpcTokenAccountBalance>,
```

Comment:
The removed response type seems to be the correct one (by looking at the [docs](https://solana.com/docs/rpc/http/gettokenlargestaccounts#response)):

```json
{
  "jsonrpc": "2.0",
  "result": {
    "context": {
      "slot": 1114
    },
    "value": [
      {
        "address": "FYjHNoFtSQ5uijKrZFyYAxvEr87hsKXkXcxkcmkBAf4r",
        "amount": "771",
        "decimals": 2,
        "uiAmount": 7.71,
        "uiAmountString": "7.71"
      },
      {
        "address": "BnsywxTcaYeNUtzrPxQUvzAWxfzZe3ZLUJ4wMMuLESnu",
        "amount": "229",
        "decimals": 2,
        "uiAmount": 2.29,
        "uiAmountString": "2.29"
      }
    ]
  },
  "id": 1
}
```

What's the reason for the change?